.github/scripts/ci.sc

```diff
@@ -219,7 +219,7 @@ def runTests(jobs: String, resultDir: Option[os.Path]) = {
       System.err.println(s"[${i+1}/${totalJobs.length}] Running test case $config,$caseName,$runCfg")
       val handle = os
         .proc("scripts/run-test.py", "-c", config, "-r", runCfg, "--no-console-log", "--base-out-dir", testRunDir, caseName)
-        .call(check=false)
+        .call(check=false, stdout=os.Path("/dev/null"))
```

Comment:
Is there anything printed to stdout?

Reply:
Those Python script logs that make it hard to read which test failed.

Reply:
Well, but log goes to stderr